frontend/packages/kubevirt-plugin/src/components/modals/disk-modal/disk-modal.tsx

```diff
@@ -202,7 +202,7 @@ export const DiskModal = withHandlePromise((props: DiskModalProps) => {
   if (source.requiresNewPVC()) {
     resultPersistentVolumeClaim = new PersistentVolumeClaimWrapper()
       .init({
-        name,
+        name: resultDataVolumeName,
```

Comment:
we don't need the generated name for the new PVCs. These PVCs will be created by the wizard and the flow will fail fast. The user then has the chance to fix the name and redo it.

Reply:
Ah, ok. Thanks for the explanation. It's helpful. Fixed!